etc/config.sample.toml

```diff
@@ -525,3 +525,26 @@
 
   # interval for how often continuous queries will be checked if they need to run
   # run-interval = "1s"
+
+###
+### [tls]
+###
+### Global configuration settings for TLS in InfluxDB.
+###
+
+[tls]
+  # Determines the available set of cipher suites. See https://golang.org/pkg/crypto/tls/#pkg-constants
+  # for a list of available ciphers, which depends on the version of Go. If not specified, uses
+  # the default settings from crypto/tls.
```

Comment:
"... from **Go's** crypto/tls **package**"? Otherwise readers may be confused about what crypto/tls is. I wouldn't expect them to see that in the URL above. Should make the same change on the next two comment blocks too.